.github/workflows/build.yml

```
@@ -174,6 +174,29 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        architecture:
+          - aarch64
+          - amd64
+        exclude:
+          # The Docker images we use for these platforms do not
+          # support aarch64.
+          - architecture: aarch64
+            platform: amazonlinux2023-systemd
+          - architecture: aarch64
+            platform: fedora38-systemd
+          - architecture: aarch64
+            platform: fedora39-systemd
+        platform:
+          - amazonlinux2023-systemd
+          - debian10-systemd
+          - debian11-systemd
+          - debian12-systemd
+          - debian13-systemd
+          - kali-systemd
+          - fedora38-systemd
+          - fedora39-systemd
+          - ubuntu-20-systemd
+          - ubuntu-22-systemd
```

Probably still should alphabetize here, even though I understand why you put kali where you did:
```suggestion
          - amazonlinux2023-systemd
          - debian10-systemd
          - debian11-systemd
          - debian12-systemd
          - debian13-systemd
          - fedora38-systemd
          - fedora39-systemd
          - kali-systemd
          - ubuntu-20-systemd
          - ubuntu-22-systemd
```